README.md

```diff
@@ -43,12 +44,23 @@ Option 1: Install from PyPI.
 $ pip install netutils
 ```
 
-Option 2: Install from a GitHub branch, such as develop as shown below.
+Option 2: Install from PyPI with optional requirements.
+
+```bash
+$ pip install netutils[napalm]
```

Comment:
Something like 

```
'''
$ pip install netutils
'''

Install with optional dependencies.

'''
$ pip install netutils[napalm]
'''
```

Please note I used single quotes instead of back ticks in example, but would expect that you would use back ticks in real implementation.